apps/video/style/video.css

```diff
@@ -443,21 +442,21 @@ header .icon:after {
 /* video player controls */
 #videoControls {
   position: absolute;
+  display: block;
```

Comment:
TBD as to whether this change is necessary based on whether 'hide-controls' element is necessary.